helm/fiaas-deploy-daemon/templates/deployment.yaml

```diff
@@ -82,3 +82,19 @@ spec:
           - mountPath: /var/run/config/fiaas/
             name: fiaas-deploy-daemon-config
             readOnly: true
+      {{- if .Values.deployment.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.deployment.nodeSelector | indent 8 }}
```

Comment:
This is a bit nitpicky but no other parts of the file move these all the way to the left. 
Would you mind changing them to look like this:
```
      nodeSelector: {{ toYaml .Values.deployment.nodeSelector | nindent 8 }}
```
?